docs/source/starter/installation.rst

```diff
@@ -0,0 +1,71 @@
+.. _installation:
+
+############
+Installation
+############
+
+--------------
+
+******************
+Lightning Coverage
+******************
+
+PyTorch Lightning is maintained and tested on different Pytorch and PyTorch versions.
```

Comment:
```suggestion
PyTorch Lightning is maintained and tested on different PyTorch versions.
```

Reply:
ah! should be Python.. my fingers have learnt `Pytorch` word. 😅

Reply:
Please help my eyes are bleeding. 
All the people spelling PyTorch wrong...